WorkloadManagementSystem/Agent/StalledJobAgent.py

```diff
@@ -115,6 +117,10 @@ def __markStalledJobs( self, stalledTime ):
 # jobs = jobs[:10] #for debugging
     for job in jobs:
       result = self.__getStalledJob( job, stalledTime )
+      site = self.jobDB.getJobAttribute( job, 'site' )['Value']
```

Comment:
Please, few things:
- make this check not here, but in function `__updateJobStatus`, so it's less heavy
- change to:
```
siteRes = self.jobDB.getJobAttribute( job, 'site' )
if not siteRes['OK']:
  return siteRes
site = siteRes
```